app/src/main/java/app/revanced/manager/ui/screen/PatchesSelectorScreen.kt

```diff
@@ -375,7 +444,7 @@ fun PatchesSelectorScreen(
                         patchList(
                             uid = bundle.uid,
                             patches = bundle.unsupported,
-                            filterFlag = SHOW_UNSUPPORTED,
+                            visible = vm.filter and SHOW_SUPPORTED == 0,
```

Comment:
I think `HIDE_UNSUPPORTED` would be more accurate, since supported patches are shown regardless of the flag.

Reply:
How would you implement that? Unless I change the filter logic in the ViewModel, HIDE_UNSUPPORTED is derived from SHOW_UNSUPPORTED.

Reply:
You can just change the name of the constant in the viewmodels companion object. The viewmodel doesn't assume anything about what the flag actually does.

Reply:
I'd also have to flip one the filter chips at line 150.

From 
```kt
CheckedFilterChip(
    selected = vm.filter and SHOW_SUPPORTED != 0,
    onClick = { vm.toggleFlag(SHOW_SUPPORTED) },
    label = { Text(stringResource(R.string.supported)) }
)
```

To

```kt
CheckedFilterChip(
    selected = vm.filter and HIDE_UNSUPPORTED == 0,
    onClick = { vm.toggleFlag(HIDE_UNSUPPORTED) },
    label = { Text(stringResource(R.string.supported)) }
)
```

Reply:
Hmm, in that case I guess the flag can be named `SHOW_UNSUPPORTED`. The filter chip should match the behavior of the flag though. It seems weird to have a flag named `SHOW_SUPPORTED` controlling visibility of the unsupported patches or a chip called "supported" that only affects the unsupported patches.

Reply:
Updated the constant name, see if the change makes sense